nae-user-ce/src/main/java/com/netgrif/application/engine/auth/service/UserServiceImpl.java

```diff
@@ -246,7 +251,15 @@ public void addAllRolesToAdminByUsername(String username) {
             throw new IllegalArgumentException("Admin user with username [%s] cannot be found.".formatted(username));
         }
         IUser user = userOptional.get();
-        user.setProcessRoles(new HashSet<>(processRoleService.findAll()));
+
+        Pageable pageable = PageRequest.of(0, paginationProperties.getBackendPageSize());
+        Page<ProcessRole> processRoles;
+        do {
+            processRoles = processRoleService.findAll(pageable);
```

Comment:
Why are we using here `do-while` with restricted page size instead of `unpaged` and one call? Could it be a performance issue to use `unpaged`? If yes, then what is the situation in case when user holds too much roles and then it is saved to database?